test/rules/comment.spec.ts

```diff
@@ -1,7 +1,7 @@
 import { stripIndent } from 'common-tags';
 import { fromFixture } from 'eslint-etc';
-import rule, { defaultOptions, ruleName } from '../../src/rules/comment';
-import { ruleTester } from '../utils';
+import rule, { defaultOptions, ruleName } from '../../src/rules/comment-syntax';
```

Comment:
Turns out the test didn't pass when I ran them locally without this change. :sweat_smile:

Reply:
Oh! good catch, ups.

Reply:
I think I changed the name at some point and prolly forgot to change this here. We should start running tests on CI via GitHub Actions I think. We can do this in another PR.